src/components/patterns/link.tsx

```diff
@@ -1,15 +1,15 @@
 import Link from "@/components/patterns/elements/drupal-link";
 import {ChevronRightIcon} from "@heroicons/react/20/solid";
 import {PropsWithChildren} from "react";
-import {twMerge} from "tailwind-merge";
+import {twJoin, twMerge} from "tailwind-merge";
 import {Maybe} from "@/lib/gql/__generated__/drupal.d";
 
 export const DrupalLinkButton = ({href, children, className = '', ...props}: PropsWithChildren<{href: string, className?: string}>) => {
   return (
     <Link
       href={href}
       {...props}
-      className={twMerge("rounded-full cta-button font-semibold leading-display block w-fit no-underline hocus:underline group transition-colors px-26 pt-10 pb-11 text-16 md:text-20 bg-digital-red hover:bg-cardinal-red-dark focus:bg-black-true active:bg-black-true text-white hocus:text-white rs-mt-neg1", className)}
+      className={twJoin("rounded-full cta-button text-16 md:text-18 font-semibold leading-display block w-fit no-underline hocus:underline group transition-colors px-26 pt-10 pb-11  bg-digital-red hover:bg-cardinal-red-dark focus:bg-black-true active:bg-black-true text-white hocus:text-white rs-mt-neg1", className)}
```

Comment:
@pookmish I tried out the `twJoin` because the `twMerge` kept dropping the `text-16` I wanted on the button for the small sizes.  Is this an ok solution? Do you have a better way? The `twJoin` keeps my `text-16` but might have other consequences I haven't found yet. I figured you'd know.

Reply:
Cool. I didn't know about that function. I have no knowledge, but if it's from the same package, I'm sure it's ok.